docs/standard/microservices-architecture/net-core-net-framework-containers/container-framework-choice-factors.md

```diff
@@ -43,17 +43,17 @@ There are several features of your application that affect your decision. You sh
     - Your .NET implementation choice is *.NET Framework* based on framework dependency.
     - Your container platform choice must be *Windows containers* because of the .NET Framework dependency.
 * Your application uses **SignalR services**.
-    - Your .NET implementation choice is *.NET Framework*, or *.NET Core (future release)*.
-    - Your container platform choice must be *Windows containers* because of the .NET Framework dependency.
-    - When **SignalR services** run on *.NET Core*, you can also choose *Linux containers*.
+    - Your .NET implementation choice is *.NET Framework*, or *.NET Core 2.1 or beyond*.
```

Comment:
or beyond -> or later versions 

should you also point out that this version is not available yet?

Reply:
ok, fixed.